kratos/utilities/constraint_utilities.cpp

```diff
@@ -0,0 +1,92 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/constraint_utilities.h"
+
+namespace Kratos
+{
+namespace ConstraintUtilities
+{
+void ResetSlaveDofs(ModelPart& rModelPart)
```

Comment:
Here you called it `ResetSlaveDofs` which I think sounds a lot more natural
@loumalouomega @RiccardoRossi @KratosMultiphysics/technical-committee are we sure we want to stick with `ResetSlave` in #4163 

Once merged there will most likely be no way back ...

Reply:
#4163 already has the name changed. allright to me

Reply:
yes is resolved